app/views/admin/wise_transfer_process.html.erb

```diff
@@ -0,0 +1,231 @@
+    <fieldset class="p-6">
+      <legend style="padding: 0px 8px">Approve</legend>
+      <p>For this approval check, you're only looking at the transfer purpose and documentation. The balance check will be done after it is approved.</p>
+      <%= button_to "💸 Approve and self-assign", approve_wise_transfer_path(@wise_transfer), method: :post %>
+
+    </fieldset>
+
+    <fieldset class="p-6">
+      <legend style="padding: 0px 8px">Reject</legend>
+      <%= form_with(model: nil, local: true, url: reject_wise_transfer_path(@wise_transfer), method: :post) do |form| %>
+        <div class="field">
+          <%= form.label "Reject with a comment", class: "bold mb1" %> <br>
+          <%= form.text_area :comment, style: "width: 400px;", placeholder: "(Markdown supported)" %>
+        </div>
+        <%= form.submit "Reject", data: { confirm: "Mark as rejected? This requires you to communicate to the organizer about the reason why." } %>
+        <small>(This requires you to communicate to the organizer about the reason why.)</small>
+      <% end %>
+
+    </fieldset>
+  <% elsif @wise_transfer.approved? %>
+    <fieldset class="p-6">
+      <legend style="padding: 0px 8px">Send</legend>
+        <% if @wise_transfer.usd_amount_cents.present? %>
+          <div class="flex flex-row justify-start items-center gap-4 mb-3">
+            <div class="field w-min-content">
+              <%= label_tag :usd_amount, "Amount (USD)", class: "bold mb1" %> <br>
+              $ <%= number_field_tag :usd_amount, render_money(@wise_transfer.usd_amount_cents, { unit: "" }),
+                placeholder: "500.00",
+                step: 0.01,
+                disabled: true %>
+            </div>
+            <% if ::Shared::AmpleBalance.ample_balance?(@wise_transfer.usd_amount_cents, @wise_transfer.event) %>
+              <p class="m-0 text-green bold flex flex-row justify-start items-center gap-2">
+                <%= inline_icon "checkmark" %>
+                Ample balance available.
+              </p>
+            <% else %>
+              <p class="m-0 text-red bold flex flex-row justify-start items-center gap-2">
+                <%= inline_icon "important" %>
+                Insufficient funds to transfer this amount.
+              </p>
+            <% end %>
+          </div>
+        <% else %>
+          <%= form_with(model: @wise_transfer, local: true, url: wise_transfer_path(@wise_transfer), method: :patch) do |form| %>
+            <div class="field">
+              <%= form.label :usd_amount, "Amount (USD)", class: "bold mb1" %> <br>
+              $ <%= form.number_field :usd_amount,
+                placeholder: "500.00",
+                step: 0.01,
+                required: true,
+                data: { controller: "truncate-decimal", action: "truncate-decimal#truncate blur->truncate-decimal#pad" } %>
+            </div>
+            <%= form.submit "🏦 Check balance" %>
+          <% end %>
+        <% end %>
```

Comment:
Is there a reason we aren't doing the balance check on the frontend using Alpine?